app/assets/stylesheets/nav.scss

```diff
@@ -21,36 +48,75 @@ $button-color: #1E90FF;
 
   input {
     width:280px;
+    border-radius: 5%;
   }
+
   label {
       color: orange;
       font-weight: bold;
+
+      span{
+        color: $purple;
+        font-size: 10px;
+      }
+
     }
+}
 
+.block {
+  display: inline-block;
 }
 
 .form {
   margin-left: 10%;
+  display: inline-block;
+}
+
+.centered_bar {
+  margin-top: 10em;
+  margin-left: 10%;
+}
 
+.home_text{
+  text-align: center;
+  color: $white;
+  font-size: 20px;
 }
 
 .inline_search{
   display: inline-block;
+
   input {
-    color: white;
+    color: $white;
     background-color: $button-color;
     border-radius: 10%;
     border: transparent;
     &:hover {
       background-color: darken($button-color, 30%);
     }
+
   }
 }
 
-.text_field {
-  input {
-    height: 5000px;
-    width: 3000px;
+.job_deets{
```

Comment:
Selector `job_deets` should be written in lowercase with hyphens<br>Opening curly brace `{` should be preceded by one space